### DISCUSSION TITLE
Firebase Authentication Error with signinwithpopup

### DISCUSSION THREAD

**Discussion by thedigitalmenagerie** [nss-evening-cohort-14/evening-client-side#184] [Help Me!] [answered]
Honey-Rae Swan 

Have you searched the other help tickets? 
Yep!

Describe your issue
I have deployed with the webpack specific commands and You can only see the background that was applied to the body in the netlify application. 

List any error messages that appear
DevTools failed to load SourceMap: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/intercom-link-expand-loader.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
https://stackoverflow.com/questions/61767538/devtools-failed-to-load-sourcemap-for-webpack-node-modules-js-map-http-e
https://answers.netlify.com/t/cdn-asset-caching-problem-after-deployment/15892
https://www.netlify.com/blog/2018/11/28/authentication-for-the-rest-of-us/

What have you tried? Tell us everything you have tried.
I am not really sure what to try to resolve this issue. 

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰
Github:
https://github.com/thedigitalmenagerie/ASSIGNMENT-Firebase-Pinterest
Netlify:
https://hr-firebase-pinterest.netlify.app/

**Comment by drteresavasquez** [comment]
Are all of your images links right? Did you add Netlify to your FB settings?

**Reply by thedigitalmenagerie** [reply]
Thanks, Dr. T! Lindsey shared the updated webpack instructions with me, so I deleted the deploy and created a new deploy with the key value pairs from the .env files included. Also, the errors had to do with some devtools settings I had to change and nothing to do with what was going wrong with the deploy on Netlify!

**Comment by thedigitalmenagerie** [comment, marked as answer]
Thanks, Dr. T! Lindsey shared the updated webpack instructions with me, so I deleted the deploy and created a new deploy with the key value pairs from the .env files included. Also, the errors had to do with some devtools settings I had to change and nothing to do with what was going wrong with the deploy on Netlify!

**Reply by jimguistical** [reply]
Thank  you for this ticket!  I had to delete the initial deployment and go back thru all the steps for it work.

**Comment by lindseysatterfield** [comment]
I was having an issue with my Google log in on my deployed site. The pop up to sign in would never fully load and then go away. I did all the steps for deployment and was unsure of what the issue was. I finally found out that when I added my key value pairs from my `.env` file to the environmental variables on Netlify, I needed to remove the quotes from the values. After I removed the quotes, I was able to log in to my deployed site. So hopefully this will help someone else if they are having the similar issue.
